### PR TITLE
Added bxt_disable_changelevel

### PR DESCRIPTION
**Pull request by SmileyAG**
This works on Windows, but I have no idea about Linux, can someone confirm it? @chinese-soup

### CODE REVIEW

File: BunnymodXT/modules/ServerDLL.cpp [hunk 1]
```diff
@@ -1092,6 +1117,8 @@ void ServerDLL::RegisterCVarsAndCommands()
 	}
 	if (ORIG_CTriggerSave__SaveTouch)
 		REG(bxt_disable_autosave);
+	if (ORIG_CChangeLevel__ChangeLevelNow)
```
Comment (hobokenn) [inline BunnymodXT/modules/ServerDLL.cpp:1120]: if (ORIG_CChangeLevel__ChangeLevelNow || ORIG_CChangeLevel__ChangeLevelNow_Linux)

**Review by hobokenn — CHANGES_REQUESTED** [review]

File: BunnymodXT/modules/ServerDLL.cpp (hunk 1 above)
Comment (hobokenn) [reply BunnymodXT/modules/ServerDLL.cpp:1120]: could you add that to SaveTouch too please?

Comment (SmileyAG) [reply BunnymodXT/modules/ServerDLL.cpp]: Done. Thanks!

**Comment by YaLTeR** [comment]
On Linux the function seems to be found but bxt_disable_changelevel 1 doesn't seem to do anything.

**Comment by YaLTeR** [comment]
Thanks

**Pull request merged**